sdk/nodejs/npm/workspaces_test.go

```diff
@@ -36,3 +36,12 @@ func TestFindWorkspaceRootNotInWorkspace(t *testing.T) {
 
 	require.ErrorIs(t, err, ErrNotInWorkspace)
 }
+
+func TestFindWorkspaceRootYarnExtended(t *testing.T) {
```

Comment:
This test is unrelated to the work, but I noticed we didn't have a test for this here when I looked at code coverage on current master.